src/styles/_mixins.scss

```diff
@@ -0,0 +1,3 @@
+@mixin columns($count) {
+  grid-template-columns: repeat($count, minmax($size, 1fr));
+}
```

Comment:
redundant. 
Why do you need mixin which includes one prop? Name columns is better than grid-template-columns?